File: ASP.NET/BoardDemo/BoardDemo/Models/Entity/Replies.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace BoardDemo.Models
{
    public class Replies
    {
        public Int64 rid { get; set; }
        [StringLength(500)]
        public string content { get; set; }
        [StringLength(100)]
        public string writer { get; set; }
        public DateTime regDate { get; set; }
        public Int64 recomCnt { get; set; }
        public Int64 foreignBoard { get; set; }
        public Int64 groupId { get; set; }
        public Int64 pid { get; set; }
    }
}

File: spring공부/BoardDemoSpringboot/src/main/java/com/demo/domain/Replies.java
package com.demo.domain;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Replies {
	private long rid;
	private String content;
	private String writer;
	private Date regDate;
	private long recomCnt;
	private long fbid;
	private long groupId;
	private long pid;
}
